src/pool.c

```diff
@@ -1537,6 +1537,54 @@ rpc_on_block_template(const char* data, rpc_callback_t *callback)
     json_object_put(root);
 }
 
+static int
+startup_scan_round_shares()
+{
+    int rc;
+    char *err;
+    MDB_txn *txn;
+    MDB_cursor *cursor;
+
+    if ((rc = mdb_txn_begin(env, NULL, MDB_RDONLY, &txn)) != 0)
+    {
+        err = mdb_strerror(rc);
+        log_error("%s", err);
+        return rc;
+    }
+    if ((rc = mdb_cursor_open(txn, db_shares, &cursor)) != 0)
+    {
+        err = mdb_strerror(rc);
+        log_error("%s", err);
+        mdb_txn_abort(txn);
+        return rc;
+    }
+    MDB_cursor_op op = MDB_LAST;
+    while (1)
+    {
+        MDB_val key;
+        MDB_val val;
+        rc = mdb_cursor_get(cursor, &key, &val, op);
+        if (rc != 0 && rc != MDB_NOTFOUND)
+        {
+            err = mdb_strerror(rc);
+            log_error("%s", err);
+            break;
+        }
+        if (rc == MDB_NOTFOUND)
+            break;
+        op = MDB_PREV;
+        share_t *share = (share_t*)val.mv_data;
+        if (share->timestamp > pool_stats.last_block_found){
+            pool_stats.round_hashes += share->difficulty;
+        } else {
+            break;
+        }
```

Comment:
Please keep formatting (brace placement and spacing) consistent with the rest of the code.

Reply:
i.e. braces on newlines or optionally single if statements no braces.

Reply:
Removed braces.